### ISSUE TITLE
Mit E-Mail registrieren

### ISSUE THREAD

**Issue by DerDast3** [haumacher/phoneblock#83]
This does not work
[Mit E-Mail registrieren](https://phoneblock.net/phoneblock/signup.jsp#registerForm)
What a pitty

**Comment by DerDast3** [comment]
it does not work on Windows 11 / Chrome
it works on iPhone

**Comment by haumacher** [comment]
Can you give more details how it "does not work"?

**Comment by CaptainWE** [comment]
@haumacher the email registration page reports "HTTP Status 500 – Internal Server Error".  I also tested with MacOS and on an iPhone, with Safari and Firefox each.

```
_An error occurred at line: [29] in the jsp file: [/signup.jsp]
PhoneBlockConfigFactory.GOOGLE_CLIENT cannot be resolved to a type
26: </div>
27: 
28: <nav class="panel">
29: 	<p class="panel-heading"><a href="<%=request.getContextPath()%>/oauth/login?force_client=<%=PhoneBlockConfigFactory.GOOGLE_CLIENT%>"><i class="fa-brands fa-google"></i> <span>Mit Google registrieren</span></a></p>
30: </nav>
31: 
32: <%_
```
 
I also added a screenshot. Would be great if you could fix this, I very much would like to register via email.
![Screenshot 2024-10-01 at 18-01-01 HTTP Status 500 – Internal Server Error](https://github.com/user-attachments/assets/19ddc46b-cf72-4f6c-a789-1fc6ca843d13)

**Comment by haumacher** [comment]
Sorry for this, I broke the registration in a last-minute change. The problem should be fixed now (01a71809a2d4c879170bb086274996267a30fcb1). @CaptainWE can you please confirm that the registration now works again for you?
